plugins/catalog-backend/src/service/NextCatalogBuilder.ts

```diff
@@ -376,7 +378,11 @@ export class NextCatalogBuilder {
       parser,
       policy,
     });
-    const entitiesCatalog = new NextEntitiesCatalog(dbClient);
+    const entitiesCatalog = new AuthorizedEntitiesCatalog(
```

Comment:
Since the catalog now has authorization built in, I think we'll need to update the [getEntityResource method in NextRouter](https://github.com/backstage/backstage/blob/master/plugins/catalog-backend/src/service/NextRouter.ts#L201) to skip authorization somehow. We could generate and supply a server token, but that seems _slightly_ weird since it's not a backend-to-backend request, but a call internal to the catalog-backend. Another option might be to expose a parameter in the catalog that disables authorization somehow on a per-call basis.

Reply:
What if we have the permission integration is created here rather than in the router? We'd simply start passing that on instead of the `permissionRules`

Reply:
Done! Check out the newest commit.